plugins/processors/scale/README.md

```diff
@@ -0,0 +1,57 @@
+# Scale Processor Plugin
+
+The scale processor filters for a set of fields,
+and scales the respective values from an input range into
+the given output range according to this formula:
+
+$$\textnormal{result}=(\textnormal{value}-\textnormal{input\_minimum})\cdot
+\frac{(\textnormal{\textnormal{output\_maximum}}-\textnormal{output\_minimum})}
+{(\textnormal{input\_maximum}-\textnormal{input\_minimum})} +
+\textnormal{output\_minimum}$$
```

Comment:
This is giving an error when previewing.

If you put it on multiple lines, maybe better to put it in a math block with ```math